### PR TITLE
Fix issues with switch pattern statement consistency

### PR DESCRIPTION
**Pull request by Kroppeb**

### CODE REVIEW

File: src/org/jetbrains/java/decompiler/modules/decompiler/SwitchPatternMatchProcessor.java [hunk 1]
```diff
@@ -330,7 +342,35 @@ private static boolean eliminateGuardRef(SwitchStatement stat, Map<List<Exprent>
               return true;
             }
             guards.put(stat.getCaseValues().get(i), new ConstExprent(0, true, null));
-            reference.a.replaceWithEmpty();
+            Statement replaced = reference.a.replaceWithEmpty();
+            replaced.getFirstSuccessor().remove();
+            Set<StatEdge> labelEdges = stat.getParent().getLabelEdges();
+            findSuccessor:
```
Comment (SuperCoder7979) [inline src/org/jetbrains/java/decompiler/modules/decompiler/SwitchPatternMatchProcessor.java:348]: Don't use labeled sequences, rather use booleans instead

**Review by SuperCoder7979 — CHANGES_REQUESTED** [review]
Minor code style tweak

**Review by SuperCoder7979 — APPROVED** [review]
LGTM now!

**Review by l-Luna — APPROVED** [review]
lgtm!

**Pull request merged**